package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "build": "npx hardhat compile",
     "test": "npx hardhat test",
-    "coverage": "npx hardhat coverage"
+    "coverage": "npx hardhat coverage",
+    "deploy": "npx hardhat run --network"
```

Comment:
Does this command actually run the deploy script? Based on [the Hardhat documentation](https://hardhat.org/guides/scripts.html) it seems like you have to use `node` to run the script.

Can we have the `deploy` command specify the network as well? IMO it leaves less room for error. Maybe we could add two commands, such as `deploy:mainnet` and `deploy:ropsten`?

Any reason not to use [a Hardhat task](https://hardhat.org/guides/create-task.html) instead of a script? They're fairly similar, but a task can easily be used in other files such as a deployment script for your tests. Up to you to decide whether you want to use a task over a script, I'm fine either way.